src/styles/blocks/card.scss

```
@@ -0,0 +1,43 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  color: #060b35;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 33px;
```

```suggestion
  padding: 34px 16px 16px;
```
set paddings for all card instead some margins for elements in card